Sources/TidepoolKit/Internal/Extensions/UserDefaults.swift

```diff
@@ -16,6 +16,7 @@ extension UserDefaults {
 
     private enum Key: String {
         case defaultEnvironment = "org.tidepool.TidepoolKit.DefaultEnvironment"
+        case studyProductSelection = "org.tidepool.plugins.TidepoolSupport.StudyProductSelection"
```

Comment:
This is outside TidepoolKit's responsibilities; this is a general purpose library for interacting with the Tidepool SDK, and these are internal Tidepool Loop concerns. Would probably be better in the TidepoolSupport framework, and the Tidepool Loop Configurator could access this code from there.